diff --git a/src/vm/moar/ModuleLoaderVMConfig.nqp b/src/vm/moar/ModuleLoaderVMConfig.nqp
index fa6883a32..517ee3f1f 100644
--- a/src/vm/moar/ModuleLoaderVMConfig.nqp
+++ b/src/vm/moar/ModuleLoaderVMConfig.nqp
@@ -2,6 +2,7 @@ role Perl6::ModuleLoaderVMConfig {
     method vm_search_paths() {
         my @search_paths;
         @search_paths.push(nqp::getcurhllsym('$PERL6_HOME') ~ '/lib');
+        @search_paths.push(nqp::getcurhllsym('$PERL6_HOME') ~ '../nqp/lib');
         # XXX CHEAT: Goes away when we implement :from<nqp>.
         @search_paths.push(nqp::getcurhllsym('$NQP_HOME') ~ '/lib');
         @search_paths
