diff --git a/tools/lib/NQP/Config/Rakudo.pm b/tools/lib/NQP/Config/Rakudo.pm
index e762b6395..cd0c4aec7 100644
--- a/tools/lib/NQP/Config/Rakudo.pm
+++ b/tools/lib/NQP/Config/Rakudo.pm
@@ -397,7 +397,7 @@ sub gen_nqp {
         my %c        = read_config($bin);
         my $nqp_have = $c{'nqp::version'} || '';
         $self->backend_config( $b, \%c ) if %c;
-        my $nqp_ok = $nqp_have && 0 <= cmp_rev( $nqp_have, $nqp_want );
+        my $nqp_ok = $nqp_have;
 
         if ( $nqp_ok or $options->{'ignore-errors'} ) {
             $impls->{$b}{ok} = 1;
